scripts/asm-installer/install_asm

```diff
@@ -1848,8 +1848,9 @@ bind_user_to_cluster_admin(){
   info "Querying for core/account..."
   local GCLOUD_USER; GCLOUD_USER="$(gcloud config get-value core/account)"
   info "Binding ${GCLOUD_USER} to cluster admin role..."
+  local PREFIX="$(echo "${GCLOUD_USER}" | cut -f 1 -d @)"
```

Comment:
Please split the `local` declaration and the assignment. e.g. `local VAR; VAR=value`. Otherwise tests won't pass since the linter will block it.

Reply:
Sorry, I missed that. I made the change.